app/clients/summary_client.py

```
@@ -2,16 +2,37 @@
 GPU 요약 서버 HTTP 클라이언트.
 
 별도의 GPU 서버로 요약 요청을 전송하고 결과를 수신합니다.
+Exponential backoff 재시도 로직이 포함되어 504/502/503/429 에러 및 타임아웃을 처리합니다.
 """
 
 import logging
 from typing import List, Optional
 import httpx
+from tenacity import (
```

Flake8 (7.3.0)

[error] 'tenacity.retry' imported but unused

(F401)